lib/CallBackery/qooxdoo/callbackery/source/class/callbackery/ui/plugin/Form.js

```diff
@@ -180,7 +180,7 @@ qx.Class.define("callbackery.ui.plugin.Form", {
                 if (s.widget == 'selectBox' || s.widget == 'comboBox'){
                     this._form.setSelectBoxData(s.key,s.cfg.structure);
                 }
-                if (s.set){
+                if (s.set && 'value' in s.set){
```

Comment:
I would also like to use the set property for things other than 'value' this patch would prevent that ...

Reply:
Fixed.